heimdall2/values.yaml

```
@@ -101,19 +107,57 @@ postgresql:
     type: ClusterIP
     port: 5672
 
-  podAnnotations: {}
-
-  podSecurityContext: {}
-
-  securityContext: {}
-
-  resources: {}
-
-  nodeSelector: {}
-
-  tolerations: []
-
-  affinity: {}
+  podAnnotations: {} 
+  # Attaches metadata to the Postgre pod
```

```suggestion
  # Attaches metadata to the Postgres pod
```

people just say postgres or postgresql, please swap this out everywhere